app/api/attendees.py

```diff
@@ -52,14 +53,26 @@ class Meta:
                          schema='EventSchema',
                          type_='event')
 
-class AttendeeList(ResourceList):
+
+class AttendeeListPost(ResourceList):
     """
-    List and create Attendees
+       List and create Attendees through direct URL
     """
 
+    @jwt_required
     def before_post(self, args, kwargs, data):
-        require_relationship(['event'], data)
+        require_relationship(['ticket', 'event'], data)
```

Comment:
This should have organizer access for POST. Please add that ([permissions](https://github.com/fossasia/open-event/blob/master/dev-handbook/resources-permissions.md#attendees---eg-v1attendees)).